specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -817,6 +817,10 @@ components:
           $ref: "#/components/schemas/TypeOppervlak"
         perceelnummerVerschuiving:
           $ref: "#/components/schemas/TypePerceelnummerVerschuiving"
+        adressen:
```

Comment:
ik zou de array wrappen in een eigen component 'LocatieKadastraalObjectCollection'. Anders krijg je een gegenereerde class met naam Adressen waarvan de type 'LocatieKadastraalObject' lijst is

Reply:
Met de feedback van Frank is de conclusie dat dit geen array hoeft te zijn.

Reply:
> Met de feedback van Frank is de conclusie dat dit geen array hoeft te zijn.

Waarmee jij erop vertrouwt dat ik beter ben in UML lezen dan jij? ... riskant

@HildeVos @rhengeveld klopt het dat er altijd 1 adres (locatie kadastraal object) is bij een kadastraal object?

Reply:
Nee, er kan ook geen adres zijn en er kunnen ook meerdere adressen zijn bij 1 object.
![image](https://user-images.githubusercontent.com/46921689/85393399-37b5ec00-b54d-11ea-8e4d-761ad6d93d67.png)

Reply:
@JohanBoer ik ben bang dat ik je dus op het verkeerde been heb gezet. Ik kan geen UML lezen...

Adres is dus toch een array.

Als ik het zo lees is er voor 1 kadastraal object 1 koppelingswijze, maar kunnen er daarbij wel meerdere adressen (objectlocatie binnenland) zijn.

Dus ik denk:
Kadastraal object ---> [1] LocatieKadastraalObject ---> [0..*] Adres

@HildeVos zie ik het zo wel goed?

@JohanBoer @MelvLee @CathyDingemanse moeten we deze complexiteit ook overnemen?

```
"locatie": {
    "koppelingswijze": {
      "code": "ADM_GEO",
      "waarde": "administratief en geometrisch"
    },
    "adressen": [
      {
        "nummeraanduidingIdentificatie": "1234567890123456"
      }
    ]
  },
```

Reply:
> ik zou de array wrappen in een eigen component 'LocatieKadastraalObjectCollection'. Anders krijg je een gegenereerde class met naam Adressen waarvan de type 'LocatieKadastraalObject' lijst is

Ik heb dit op basis van deze reactie van Melvin gedaan.